examples/loader-snowflake/src/data/api-requests.csv.ts

```diff
@@ -0,0 +1,21 @@
+import {csvFormat} from "d3-dsv";
+import {utcDay} from "d3-time";
+import {run} from "./snowflake.js";
+
+const end = utcDay();
+const start = utcDay.offset(end, -1);
+
+process.stdout.write(
+  csvFormat(
+    await run((query) =>
+      query(
+        `SELECT normalized_path AS "path", COUNT(*) AS "count"
```

Comment:
I'm a little torn on specifying explicitly lowercase column names here. on one hand it plays better with downstream usage but on the other it can lead to a lot of tedious repetition in sql queries here. I guess I'm just hoping people won't cargo cult it when it might not apply to them.

Reply:
The point here is the data loader controls what the output shape is. It shouldn’t be a requirement that the data loader passes through the column names as-is.